<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErosionEditor</class>
 <widget class="QWidget" name="ErosionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="LabelSlider" name="densityLabel">
     <property name="text">
      <string>Density:</string>
     </property>
     <property name="buddy">
      <cstring>densityField</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="densityField">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabelSlider" name="strengthLabel">
     <property name="text">
      <string>Strength:</string>
     </property>
     <property name="buddy">
      <cstring>densityField</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="strengthField">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelSlider</class>
   <extends>QLabel</extends>
   <header>labelslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
